src/libraries/Native/Unix/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -49,10 +49,10 @@ set(NATIVECRYPTO_SOURCES
     pal_x509ext.c
 )
 
-# Always build portable on macOS because OpenSSL is not a system component
+# Always build portable on macOS/iOS because OpenSSL is not a system component
```

Comment:
Honestly, we should probably just not build this component when building for iOS.

Reply:
Why not? Is it because of the API differences? If yes then there's a need for some plan and tracking issue to fix that. Last time I looked at it the basic crypto was already quite unified across the two OSes, the x509 handling was more limited on iOS though.

Reply:
This particular component is for interacting with OpenSSL.  I don't think we need RSAOpenSsl and other OpenSSL P/Invoke cooperation to work on iOS.

Reply:
Ah, my bad. I was reading it on mobile GitHub version and missed that it is not the `.Apple` one.

Reply:
Yeah I think it makes sense to disable this one.